ui/src/Components/ChatHeader.js

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
+import ChatHeaderRecipient from './ChatHeaderRecipient';
 
 const ChatHeader = ({msg}) => {
-  return <div class="header-container">{msg.subject}</div>;
+  return (
+    <div class="header-container">
+      {msg.subject}
+      <div class="component-header">
```

Comment:
is this class required? the component header was used in the thread component.

Reply:
I have removed component-header div